lib/hawkular_monkey_patches.rb

```diff
@@ -13,23 +13,26 @@ def children_domain_controllers(recursive = false)
         def children_domain_hosts(recursive = false)
           children_domain_hosts = []
           ManageIQ::Providers::Hawkular::MiddlewareManager::SUPPORTED_VERSIONS.each do |version|
-            children_by_type("Domain Host #{version}", recursive)
+            children_domain_hosts += children_by_type("Domain Host #{version}", recursive)
```

Comment:
+1
It seems there was a bug here with the change of the supported versions.
Good catch.